visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.tsx

```diff
@@ -198,7 +221,7 @@ export class CohortEditor extends React.PureComponent<ICohortEditorProps, ICohor
             this.setState(
                 {
                     openedFilter: {
-                        arg: this.props.jointDataset.metaDict[openedFilter.column].featureRange.max,
+                        arg: [...this.state.openedFilter.arg, this.props.jointDataset.metaDict[openedFilter.column].featureRange.max],
```

Comment:
this seems like you are preserving any indexes from the categorical selection, and appending the max numeric value to the end. In all the filters, you now take the 0th element, rather than cast to number so this won't break, but it will use the incorrect element if a non-empty array is converted to a number here.

Reply:
I think it should just be [this.props.jointDataset.metaDict[openedFilter.column].featureRange.max]

Reply:
yes that right. I was trying out some other ways to implement it and never refactored this correctly!